tests/unit/test_charm.py

```diff
@@ -922,31 +920,15 @@ def test_delete_password(self, _):
         assert self.harness.charm.get_secret("unit", "somekey") is None
 
         self.harness.set_leader(True)
-        with self._caplog.at_level(logging.ERROR):
-            self.harness.charm.remove_secret("app", "replication")
-            assert (
-                "Non-existing field 'replication' was attempted to be removed" in self._caplog.text
-            )
-
-            self.harness.charm.remove_secret("unit", "somekey")
-            assert "Non-existing field 'somekey' was attempted to be removed" in self._caplog.text
-
-            self.harness.charm.remove_secret("app", "non-existing-secret")
-            assert (
-                "Non-existing field 'non-existing-secret' was attempted to be removed"
-                in self._caplog.text
-            )
-
-            self.harness.charm.remove_secret("unit", "non-existing-secret")
-            assert (
-                "Non-existing field 'non-existing-secret' was attempted to be removed"
-                in self._caplog.text
-            )
```

Comment:
This must stay, with caplog.DEBUG. Exactly as done in https://github.com/canonical/pgbouncer-operator/pull/207/files#diff-0580ab9aef31b44580478eb270ec3422c5e6fef35df585fecb3a68317e246809L639